src/main.tsx

```diff
@@ -0,0 +1,13 @@
+import { StrictMode } from 'react';
+import { createRoot } from 'react-dom/client';
+import './index.scss';
+import App from './App.tsx';
+import ErrorBoundary from './components/ErrorBoundary.tsx';
+
```

Comment:
add for fix eslint error
```suggestion

// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```